trade-finance-manager-api/src/v1/validation/route-validators/route-validators.js

```diff
@@ -21,13 +21,13 @@ const bankIdValidation = param('bankId')
 
 const mongoIdValidation = param('_id').isMongoId().withMessage("Invalid MongoDB '_id' path param provided");
 
+const sqlIdValidation = param('id').isInt().withMessage("Invalid 'id' path param provided");
```

Comment:
Are we expecting these to always be positive? If so we could do something like
```suggestion
const sqlIdValidation = param('id').isInt({min: 0}).withMessage("Invalid 'id' path param provided");
```
just to tie it down even further